Tests/FoundationTests/DateExtensionsTests.swift

```diff
@@ -459,7 +459,7 @@ final class DateExtensionsTests: XCTestCase {
     }
 
     func testTimezone() {
-        XCTAssertEqual(Date().timeZone, Calendar.current.timeZone)
+        XCTAssertEqual(NSTimeZone.default, Calendar.current.timeZone)
```

Comment:
Please revert this as it's testing the actual extension.
```suggestion
        XCTAssertEqual(Date().timeZone, Calendar.current.timeZone)
```